markdown/events/2022/bytesize_nf-core-airrflow.md

```diff
@@ -24,3 +24,87 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[9:01](https://www.youtube.com/watch?v=CrJgxVRVlqY&t=541)
+Starting for the QCN sequence assembly, the pipeline supports different sequencing protocols, including multiplex PCR, in which the users have to provide the B and C-primer sequences that were used for the amplification, or in the case of 5' RACE, providing the C-primer and the linker sequences for amplification. Both protocols are supported with and without UMI barcodes. The barcodes can also be provided in different configurations. Starting from the raw sequencing data, a sample sheet needs to be provided that contains sample information and the individual FASTQ files for all samples. Depending on if the sequencing protocol includes UMI barcodes or not, there will be some processes but they all start with quality control of the reads with FASTQ, filtering the sequences by quality threshold, masking the primer sequences and if a UMI-based protocol is used, then a consensus is built from all the sequences that have the same UMI barcode. This way, it also allows to correct for the errors as I mentioned before.
+There is an extra procedure, that is employed whenever it's estimated that the length of the UMI barcodes will not be sufficient to cover all of the diversity of the sample and that is bypassed by first clustering all the sequences by similarity and annotating the cluster ID and then two different sequences with the same UMI barcode can also be distinguished this way.
+
+[10:51](https://www.youtube.com/watch?v=CrJgxVRVlqY&t=651)
+After building the consensus, all the sequences that contain the same UMI barcode are collapsed and the count, the number of sequences with the same UMI barcode, is annotated. Other metadata is also annotated as well as the count of duplicate sequences with different UMI barcodes that were collapsed. This can be useful for filtering. After this step there comes the VBJ assignment and filtering step and here it's also possible
+to start with already assembled sequences, that can be provided with a sample sheet and FASTA files. Typically single cell sequencing data processing starts at this step, because the pipeline supports directly the output from the tool 10x Genomics CellRanger multi, which provides, while incorporating also TCR and VCR sequencing in the 10x Genomics sequencing procedure, the output of that tool, which is the AIRR rearrangement
+table, thatt contains all of the sequences there. This can also be directly provided to the pipeline at that step. What that step does, it aligns the sequences to the IMGT reference and then assigns what exact B, D and J segments were used there. In the case of the single cell data there is an option. This gene reassignment step is optional.
```

Comment:
```suggestion
table, that contains all of the sequences there. This can also be directly provided to the pipeline at that step. What that step does, it aligns the sequences to the IMGT reference and then assigns what exact V, D and J segments were used there. In the case of the single cell data there is an option. This gene reassignment step is optional.
```